vulnerabilities/importers/debian_oval.py

```diff
@@ -52,35 +53,24 @@ def __init__(self, *args, **kwargs):
         # we could avoid setting translations, and have it
         # set by default in the OvalParser, but we don't yet know
         # whether all OVAL providers use the same format
-        self.translations = {'less than': '<'}
+        self.translations = {"less than": "<"}
         self.pkg_manager_api = DebianVersionAPI()
 
     def _fetch(self):
-        base_url = 'https://www.debian.org/security/oval/'
-        file_name = 'oval-definitions-{}.xml'
+        base_url = "https://www.debian.org/security/oval/"
+        file_name = "oval-definitions-{}.xml"
         releases = self.config.releases
         for release in releases:
             file_url = base_url + file_name.format(release)
```

Comment:
A single format or f-string would be better IMHO

Reply:
Right